### ISSUE TITLE
unordered continuation error at client-parser-ws.c#84 doesn't call LWS_CALLBACK_CLIENT_CONNECTION_ERROR

### ISSUE THREAD

**Issue by Ninetalesgh** [warmcat/libwebsockets#2774]
Accidentally sending \r\n\r\n\0 instead of just \r\n\r\n at the end of a handshake will have LWS correctly treat the \0 as separate packet.
That packet then is interpreted to have fin=0 and opcode=0 without having received a first fragment prior.

Would it be appropriate to upgrade the info to an error message (since it does destroy the connection) ?

**Comment by lws-team** [comment]
Yes, if I understand it sounds like it should better error out, dropping the connection should do.  Patch welcome.
